src/RmCommand.cc

```diff
@@ -101,7 +101,7 @@ static int rm(const string& trace, const RmFlags& flags, FILE* out) {
   if (!flags.force && !is_trace(trace_path)) {
     fprintf(stderr,
             "\n"
-            "rr: Could not idenfity '%s' as a trace, use -f to remove anyway"
+            "rr: Could not identity '%s' as a trace, use -f to remove anyway"
```

Comment:
This is actually supposed to be `identify`

Reply:
Thanks!  

Modified